api/src/auth/drivers/local.ts

```diff
@@ -65,6 +65,10 @@ export function createLocalAuthRouter(provider: string): Router {
 			const accountability: Accountability = {
 				ip: getIPFromReq(req),
 				role: null,
+				user: null,
+				roles: [],
+				admin: false,
+				app: false,
```

Comment:
Seeing this change (the addition of more properties to the Accountability context) in quite a few places. Would it make sense to abstract this away into a `createDefaultAccountability(ip?: string)` util, just so we have one place to change if we require more default parameters at a later point in time?